### PR TITLE
[zh] fix format mistake

### PR DESCRIPTION
**Pull request by wardenlym**
fix format mistake in zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md

### CODE REVIEW

**Comment by wardenlym** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> 📝 **Please follow instructions at https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment and we'll verify. Thanks.
> 
> * If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address.  Check your existing CLA data and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please log a ticket with the Linux Foundation Helpdesk: https://support.linuxfoundation.org/
> * Should you encounter any issues with the Linux Foundation Helpdesk, send a message to the backup e-mail support address at: [login-issues@jira.linuxfoundation.org](mailto:login-issues@jira.linuxfoundation.org)

i have signed cla

**Review by tanjunchen — COMMENTED** [review]
/lgtm
/cc @tengqm

File: content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ such as systemd.
 由于硬件、操作系统、网络或者其他主机特定参数的差异。某些主机需要特定的 kubelet 配置。
 以下列表提供了一些示例。
 
-- 由 kubelet 配置标志 `--resolv-confkubelet` 指定的 DNS 解析文件的路径在操作系统之间可能有所不同，
+- 由 kubelet 配置标志 `--resolv-conf` kubelet 指定的 DNS 解析文件的路径在操作系统之间可能有所不同，
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/kubelet-integration.md:153]:
```suggestion
- 由 kubelet 配置标志 `--resolv-conf` 指定的 DNS 解析文件的路径在操作系统之间可能有所不同，
```

**Comment by tengqm** [comment]
@wardenlym Please run the following command **on the master branch** to check if there are other changes to sync/translate.

```shell
./scripts/lsync.sh content/zh/docs/tasks/run-application/horizontal-pod-autoscale.md
```

**Pull request merged**